reflex-app/src/animations/SocialLineIcons/SocialLineIcons.js

```diff
@@ -0,0 +1,116 @@
+import React, { Fragment, Component, PureComponent } from 'react';
+import PropTypes from 'prop-types';
+import styled, { keyframes } from 'styled-components';
+
+const traceFb = keyframes`
+  0% {
+    stroke-dashoffset: 200;
+  }
+  75% {
+    stroke-dashoffset: 0;
+    fill: rgba(66,102,178,0.3);
```

Comment:
How did you get these fill values? Can you leave a comment somewhat describing how you got these?